### ISSUE TITLE
Hunter Pet Naming Griefing

### ISSUE THREAD

**Issue by ManneN1** [Macumbafeh/TotemPlatesOnyxiaTBC#2]
Hello,

Please add the possibility to add either a CreatureType check (~= Beast), an NPC ID check, or an arenapet1-5 name comparison check for the nameplates, to ensure Hunter pets cannot grief the functionality of this Addon.

Thanks for a great addon.

**Comment by Macumbafeh** [comment]
Hello, i'll check that. You get an issue as a hunter? Or it was some mobs which get icons?
Thanks for the feedback

**Comment by ManneN1** [comment]
No, the issue is enemy Hunters can name their pet the same as a totem or another tracked unit (e.g., Shadowfiend) and grief the system, making the pet nameplate into an icon. Not uncommon in arena at least.

**Comment by Macumbafeh** [comment]
Ah got it ! 👍

**Comment by Macumbafeh** [comment]
Done on this new version : https://github.com/Macumbafeh/TotemPlates_awesome_wotlk

Now it scan NPCID, so even if a pet is named like a totem or any other minion, it will not grief the system.
